src/NuGet.Core/NuGet.Common/XmlUtility.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace NuGet.Common
+{
+    public static class XmlUtility
+    {
+        /// <summary>
+        /// Creates a new System.Xml.Linq.XDocument from an System.Xml.XmlReader.
```

Comment:
I don't think the comment should not communicate the internal implementation of a method. 
Especially cause this is something that can't be enforced by the contract itself.